### DISCUSSION TITLE
Uninstall finops toolkit and redeploy under another sub and rg

### DISCUSSION THREAD

**Discussion by saleemmy** [microsoft/finops-toolkit#819] [Q&A] [answered]
Hi, I am looking for steps to uninstall the current finops toolkit and redeploy in another sub and rg.

**Comment by lundejd** [comment]
What I would do is delete the resource group that holds the FinOps toolkit you no longer need, then create a new RG in the new sub and re-deploy.

**Reply by saleemmy** [reply]
I cannot delete the rg since other resources are deployed in the same rg

**Comment by arthurclares** [comment, marked as answer]
@saleemmy You don't need to remove one instance of Hubs to deploy another. That said, if you want to remove it, the easiest option is to use Remove-FinopsHub Powershell function https://microsoft.github.io/finops-toolkit/powershell/hubs/Remove-FinOpsHub

**Reply by flanakin** [reply]
@saleemmy Does this help? 

Also note each of the deployed resources share a naming pattern, so they should be easily identifiable. The list of resources that are deployed with FinOps hubs are documented @ https://microsoft.github.io/finops-toolkit/hubs/template.
